packages/web/src/components/ControlledCustomAutocomplete/index.tsx

```diff
@@ -36,10 +36,12 @@ import { processStepWithExecutions } from 'components/PowerInput/data';
 import { StepExecutionsContext } from 'contexts/StepExecutions';
 
 interface ControlledCustomAutocompleteProps
-  extends AutocompleteProps<IFieldDropdownOption, boolean, boolean, boolean> {
+  extends Omit<
+    AutocompleteProps<IFieldDropdownOption, boolean, boolean, boolean>,
+    'renderInput'
```

Comment:
I omit "renderInput" prop here, because it is not used in the custom component, but required in the Autocomplete. It was confusing that this prop was passed but not used.